### DISCUSSION TITLE
"esp-ferment-engine"

### DISCUSSION THREAD

**Discussion by DByhlin** [jeroen79/esp-brew-engine#48] [Ideas]
Thought of making a fermentation version with iSpindel integration through iSpindel iRelay (https://www.nautilis.eu/portfolio-item/nautilis-ispindel-relay/)?

**Comment by jeroen79** [comment]
That would make a nice addition to the esp-brew-engine and esp-bottle-filler, i will consider it, but it will depend on how much free time i have.
